.github/workflows/ci-lint.yaml

```diff
@@ -48,6 +51,14 @@ jobs:
           towncrier check --compare-with origin/${BASE_BRANCH}
         env:
           BASE_BRANCH: ${{ github.base_ref }}
+        if: |
+          github.event_name == 'pull_request' &&
+          github.actor != 'dependabot[bot]'
+      - name: Lint git commits
+        run: |
+          make lint-git
+        # Always run this step so that all linting errors can be seen at once, but skip for dependabot
+        if: always() && github.actor != 'dependabot[bot]'
```

Comment:
We probably need this for some other CI jobs too? See https://github.com/oasisprotocol/nexus/pull/663 and all the jobs that fail there. But if we skip them, I guess we'll have the same problem that I was solving in https://github.com/oasisprotocol/nexus/pull/660 because required jobs can't be skipped? I hope there's a better solution to that problem than my hack in #660.

Reply:
(relates to lint workflow only) We are skipping steps in a job not a workflow, so we won't hit #660 problem here.

I think build issues will have to be handle by someone more familiar with Nexus.

Reply:
Ah the build issues in #663 actually look genuine; looks like dependabot was not fully successful with its patch.

But there's also an issue in #663 where it complains of a missing changelog fragment. That's what I meant we should create an exemption for.

Reply:
towncrier is handled in L~54

Reply:
Ah I missed that, thank you!